mautic/mautic
# Fix geolocation even if anonymize_ip is enabled

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Noticed If Anonymize IP  is enabled not geolocation provide. 
This PR fixed it. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Enable Anonymize IP in configuration > Tracking Settings
2. Setup tracking code and go to incognito window  and hit the trakcing page
3. You contact should be IP address in anonymous  contact list tab without country flag.


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat all steps
3. You contact should listed with country flag
